Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -85,7 +85,7 @@ type BocAutoCompleteReferenceValue_Options = {
     highlight(value: string, term: string): string;
     scroll: boolean;
     repositionInterval: number;
-    parse(data: BocAutoCompleteReferenceValue_Item[]): BocAutoCompleteReferenceValue_CacheRow;
+    parse(data: BocAutoCompleteReferenceValue_Item[]): Remotion.Autocompleter.BocAutoCompleteReferenceValue_CacheRow;
```

Comment:
Remotion.Autocompleter => naming needs something. `Autocompleter` just feels like it's using the wrong naming convenstions for the code base.

Reply:
What name would you like? `Remotion.AutoComplete`?

Reply:
Hmmm....the problem is that the widget as a whole is named BocAutoCompleteReferenceValue. So, when you then prefix `Remotion.AutoComplete` (e.g. right here) it looks really weird. We may need to update a few other bits as well, e.g. shortening things.

I'm thinking ComboBox, AutoComplete, AutoCompleteTextField, AutoCompleteInput, AutoCompleteHtmlElement, etc. Could you sumarize the scope of the concept that gets the name? And while I'm writing this, maybe `BocAutoCompleteBehavior` (or something along those lines with better declination) might be the best choice? As I said, it really depends on what the scope of the concept is.

Reply:
The scope here is just all the classes that are necessary for the autocomplete, we can also use just `Remotion` here and have all the other classes on there

Reply:
Discussed:
It's a namespace, not a type.
`Remotion.BocAutoCompleteReferenceValue` and the prefix `BocAutoCompleteReferenceValue_` will be removed for this.